src/meltano/core/logging/utils.py

```diff
@@ -197,6 +197,18 @@ def writeline(self, line: str):
 
 async def _write_line_writer(writer: SubprocessOutputWriter, line: bytes):
     # StreamWriters like a subprocess's stdin need special consideration
+    def _decode_log(byte_string):  # 8377
+        encodings = [
+            "latin-1"
+        ]  # list of encoding we need to consider; default is utf-8
+        for enc in encodings:
+            try:
+                return byte_string.decode(enc)
+            except UnicodeDecodeError:
+                pass
+        else:
+            return byte_string.decode("utf-8", errors="ignore")
```

Comment:
Try-except blocks are expensive, so I'd prefer to simple defer to `decode(errors="replace")` if possible. What do you think?

Reply:
As this is for logging. Losing some non-decodable character mightn't make much difference. So, yeah, having simple `decode(error="replace")` makes a block more straightforward, faster and readable. I will make a change and push it.